Framework/Algorithms/test/PolarizationCorrections/PolarizationEfficienciesWildesTest.h

```diff
@@ -294,39 +294,45 @@ class PolarizationEfficienciesWildesTest : public CxxTest::TestSuite {
   ///  Test calculations
 
   void test_all_calculations_are_correct_using_mag_ws() {
-    runCalculationTest(nullptr, nullptr, 1.03556249, 0.93515155, 1.07112498, 0.87030310);
+    runCalculationTest(nullptr, nullptr, {1.03556249, 0.1951872164}, {0.93515155, 0.1727967724},
+                       {1.07112498, 0.3903744329}, {0.87030310, 0.3455935448});
   }
 
   void test_all_calculations_are_correct_using_input_P_ws() {
-    const double expectedPEfficiency = 0.98;
-    const double expectedTPMO = (2 * expectedPEfficiency) - 1;
-    const double expectedAEfficiency = (EXPECTED_PHI / (2 * expectedTPMO)) + 0.5;
-    const double expectedTAMO = (2 * expectedAEfficiency) - 1;
+    const std::pair<double, double> expectedPEfficiency = {0.98, 0.9899494934};
```

Comment:
Another preference type situation, so feel free to leave as is, i think here compiler can infer the type without ambiguity, but as there are many `const std::pair<double,double>`,  perhaps it would make it a bit cleaner to use an alias?
```suggestion
    const std::pair expectedPEfficiency = {0.98, 0.9899494934};
```